src/cli/domain/package-server-script/bundle/config/index.js

```diff
@@ -8,6 +8,7 @@ var BabiliPlugin = require('babili-webpack-plugin');
 
 module.exports = function webpackConfigGenerator(params){
   return {
+    devtool: '#source-map',
```

Comment:
why not `cheap-module-eval-source-map`?  https://webpack.js.org/configuration/devtool/#devtool